src/App.js

```diff
@@ -13,6 +13,17 @@ export default function App() {
 
   const navigate = useNavigate();
 
+  function onEdit(updatedPlant) {
+    const newPlants = savedPlants.map(savedPlants => {
+      if (savedPlants._id === updatedPlant._id) {
+        const newPlant = { ...savedPlants, ...updatedPlant };
```

Comment:
```suggestion
        const newPlant = { ...savedPlants, updatedPlant };
```
why do you use the spread operator at updatedPlant? Isn't it just one value that is supposed to be stored in updatedPlant? the spread operator is usually used for arrays :)